frontend/spec/selenium/spec/locations_spec.rb

```diff
@@ -93,6 +93,16 @@
     @driver.find_element_with_text('//div[contains(@class, "alert-success")]', /Location Created/)
   end
 
+  it 'allows the browse locations list to be filtered by repository' do
```

Comment:
Is this intentional? This test is causing your PR to fail, and appears to be unrelated to the rest of the PR. Perhaps this was included by accident?